src/uniqWith.js

```diff
@@ -19,10 +19,10 @@ var _curry2 = require('./internal/_curry2');
  * @example
  *
  *      var strEq = R.eqBy(String);
- *      R.uniqWith(strEq)([1, '1', 2, 1]); //=> [1, 2]
- *      R.uniqWith(strEq)([{}, {}]);       //=> [{}]
- *      R.uniqWith(strEq)([1, '1', 1]);    //=> [1]
- *      R.uniqWith(strEq)(['1', 1, 1]);    //=> ['1']
+ *      R.uniqWith(strEq, [1, '1', 2, 1]); //=> [1, 2]
+ *      R.uniqWith(strEq, [{}, {}]);       //=> [{}]
+ *      R.uniqWith(strEq, [1, '1', 1]);    //=> [1]
+ *      R.uniqWith(strEq, ['1', 1, 1]);    //=> ['1']
```

Comment:
in a case like this, it might make sense to define `const uniqByStr = R.uniqWith(R.eqBy(String))` and use that in the examples